### PR TITLE
P Trace parsing and an end-to-end test

### PR DESCRIPTION
**Pull request by dijkstracula**

### CODE REVIEW

File: src/test/java/ClientServerTraceParserTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        assertEquals(1, evs.size());
+        assertEquals(
+                new PObserveEvent(0, new ClientServer.eReadQueryResp(new ClientServer.PTuple_accountId_balance(0, 15))),
+                evs.get(0));
+    }
+
+    @Test
+    @DisplayName("Can extract multiple SendLog lines with increasing timestampe")
+    public void testCanExtractMultipleSendLogLines() {
```
Comment (dijkstracula) [inline src/test/java/ClientServerTraceParserTest.java:82]: @beyazit-yalcinkaya @ankushdesai here's one of the tests that might be useful to see the structure I was envisioning, which of course we can iterate on if there are things we don't like.  A PObserveEvent is just a timestamp (which you can see increases from 0 to 1 to 2 here) and a PEvent.

**Comment by ankushdesai** [comment]
Looks great! Can we rename the PObserveEvent as PEvent. For me, a PObserveEvent is (TimeStampInterval, PEvent)

**Comment by ankushdesai** [comment]
> > For me, a PObserveEvent is (TimeStampInterval, PEvent)
> 
> Hey Ankush, yep, this is the definition I've been trying to stick with (though the timestamp is just an int at the moment) - is [this](https://github.com/dijkstracula/prtsandbox/blob/8f5ef6755ad75e82702071f4df4a1163c18a5b2d/src/main/java/prt/PObserveEvent.java) what your mental model was too?

Lets convert the int to a type TimeStampInterval (earliest and latest).

**Comment by dijkstracula** [comment]
> Lets convert the int to a type TimeStampInterval (earliest and latest).
[Done!](https://github.com/dijkstracula/prtsandbox/pull/29/commits/6cfb612b7bc74e850384cc1d3c342f94715a0c9c#diff-4a1282196a30a2054aaa88e959e7a85c4677eb899f4b40ab0780565dbf27c0d5)

**Pull request merged**